components/get_team_invite_link_modal/index.js

```diff
@@ -1,16 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I removed this entire component because is not longer used with the new Invite modal.